docs/shared/ungrib.yaml

```diff
@@ -0,0 +1,10 @@
+ungrib:
+  execution:
+    mpicmd: srun
+    batchargs:
+      nodes: 1
+      walltime: 00:01:00
```

Comment:
Is this a realistic time for this application?

Reply:
It was previously `00:05:00` and Paul suggested `00:01:00`. So far I've seen it run within `00:00:05` and `00:00:07`. Would you suggest increasing it?